src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -1,5 +1,30 @@
 [
   {
+    "name": "Data Loss Prevention",
+    "documentation": "https://cloud.google.com/dlp/docs/reference/libraries",
+    "description":"The DLP API enables clients to detect the presence of Personally Identifiable Information (PII) and other privacy-sensitive data in user-supplied, unstructured data streams such as text blocks and images.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/dlp/compute.svg",
```

Comment:
This icon doesn't exist, and actually none  of the URLs seem correct.  Perhaps the tests should check the URLs for 404s?